diff --git a/ios/App/App/NativeYouTubeViewController.swift b/ios/App/App/NativeYouTubeViewController.swift
index 1228a58e053555c4df90c624b79fab56a0fcf905..422323dee1484e9f12c695b5a95f3aabaab885c2 100644
--- a/ios/App/App/NativeYouTubeViewController.swift
+++ b/ios/App/App/NativeYouTubeViewController.swift
@@ -1,14 +1,19 @@
 import Capacitor
 import UIKit
 
 class NativeYouTubeViewController: CAPBridgeViewController {
     override func capacitorDidLoad() {
         super.capacitorDidLoad()
+#if DEBUG
+        if #available(iOS 16.4, *) {
+            webView?.isInspectable = true
+        }
+#endif
         bridge?.registerPluginInstance(NativeYouTubePlugin())
         bridge?.registerPluginInstance(NativeGoogleAuthPlugin())
         bridge?.registerPluginInstance(NativeFcmTokenPlugin())
         print("[NativeYouTubePlugin] registered")
         print("[NativeGoogleAuthPlugin] registered")
         print("[NativeFcmTokenPlugin] registered")
     }
 }
